pandas/tests/io/test_stata.py

```diff
@@ -99,9 +99,9 @@ def setup_method(self, method):
 
         self.stata_dates = os.path.join(self.dirpath, 'stata13_dates.dta')
 
-    def read_dta(self, file):
+    def read_dta(self, file, encoding='latin-1'):
```

Comment:
Should use `None` as encoding so it can be overridden by the reader depending on the dta version.